x-pack/plugins/triggers_actions_ui/public/application/components/builtin_alert_types/threshold/expression.scss

```diff
@@ -1,3 +1,13 @@
 .actAlertVisualization__chart {
-  height: $euiSize * 15;
+  height: $euiSize * 14;
+}
+
+.actAddAlertSteps {
+  .euiStep__titleWrapper {
+    align-items: center;
+  }
+
+  .euiStep__title {
+    @include euiTitle('xs');
+  }
 }
```

Comment:
If this is the only way to do this, we need to make an issue on the EUI side to make this more flexible. Make an issue on EUI for what you couldn't achieve.

Reply:
How about:

Enable alert
Mute sound or Mute audio